lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -1689,13 +1689,14 @@ def test_empty(self):
         for request in httpretty.httpretty.latest_requests:
             self.assertEqual(request.method, "GET")
 
-    def test_basic(self):
-        self.register_comment()
+    @ddt.data(None, "test_parent")
+    def test_basic(self, parent_id):
+        self.register_comment({"parent_id": parent_id})
         actual = update_comment(self.request, "test_comment", {"raw_body": "Edited body"})
         expected = {
             "id": "test_comment",
             "thread_id": "test_thread",
-            "parent_id": None,  # TODO: we can't get this without retrieving from the thread :-(
+            "parent_id": parent_id,
```

Comment:
Weird, I thought this change went through already unless there were two of these.